serverless.yml

```diff
@@ -9,12 +9,16 @@ plugins:
 provider:
   name: aws
   runtime: nodejs8.10
-  # region: ap-southeast-2
+  region: ap-southeast-2
   iamRoleStatements:
     - Effect: "Allow"
-      Action: 
-       - "cloudwatch:DescribeAlarmHistory"
-       - "cloudwatch:PutMetricData"
+      Action:
+        - "cloudwatch:DescribeAlarmHistory"
+        - "cloudwatch:PutMetricData"
+      Resource: "*"
```

Comment:
Can we restrict this at all - particularly writes to a specific namespace?

Reply:
Not necessarily because the metrics are created based on the service-name (which comes from alarm name)